.github/dependabot.yaml

```diff
@@ -0,0 +1,18 @@
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "daily"
+    commit-message:
+      prefix: "ci: ⬆️ "
+
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: "fix: ⬆️ "
```

Comment:
this is an academic question... is this a `fix`? Personally, I'd go with `feat`.

Reply:
release-please folks are using `fix`. (https://github.com/googleapis/release-please/pull/1974)

IMHO it's `build` but that is not recognized as version bump change.

It would be fix if the update would address security issue. And it does not add a new feature.